frontend/src/app/supportAdmin/ManageFacility/FacilityInformation.tsx

```diff
@@ -0,0 +1,152 @@
+    closeModal();
+  };
+
+  const confirmationModal = (
+    <Modal
+      onClose={closeModal}
+      title="Delete facility"
+      contentLabel="Confirm delete facility"
+      showModal={showModal}
+    >
+      <Modal.Header
+        styleClassNames={"font-heading-lg margin-top-0 margin-bottom-205"}
+      >
+        Delete {manageFacilityState.facility?.name}
+      </Modal.Header>
+      <div className="border-top border-base-lighter margin-x-neg-205"></div>
+      <p className="margin-top-4">
+        Deleting {manageFacilityState.facility?.name} will remove any
+        in-progress tests from the test queue. Users only added to this facility
+        will lose access to SimpleReport until an organization admin reassigns
+        them. Patients only added to this facility also need to be reassigned.
+      </p>
+      <p className="margin-top-3">
+        There are{" "}
+        <strong>{manageFacilityState.facility?.usersCount} users</strong> and{" "}
+        <strong>{manageFacilityState.facility?.patientsCount} patients</strong>{" "}
+        added only to this facility.
+      </p>
+      <div className="border-top border-base-lighter margin-x-neg-205"></div>
+      <Modal.Footer
+        styleClassNames={"display-flex flex-justify-end margin-top-205"}
+      >
+        <Button
+          className="margin-right-205"
+          variant="unstyled"
+          label="No, go back"
+          onClick={closeModal}
+        />
+        <Button
+          className="margin-right-0"
+          label="Yes, delete facility"
+          onClick={handleConfirmDeletion}
+        />
+      </Modal.Footer>
+    </Modal>
+  );
+
+  /**
+   * HTML
+   */
+  return (
+    <div className="prime-container card-container" aria-live={"polite"}>
+      {manageFacilityState.facility ? (
+        <>
+          <div className="usa-card__header">
+            <h2 className="desktop:grid-col-fill tablet:grid-col-fill mobile:grid-col-12 font-heading-lg margin-top-0 margin-bottom-0">
+              {manageFacilityState.facility.name}
+            </h2>
+          </div>
+          <div className="usa-card__body">
+            <h3 className="margin-y-3">Facility information</h3>
+            <FacilityDataItem
+              title="Facility name"
+              description={manageFacilityState.facility.name}
+            />
+            <FacilityDataItem
+              title="Location"
+              description={`${
+                manageFacilityState.facility.city !== "" &&
+                manageFacilityState.facility.city?.concat(", ")
+              }${manageFacilityState.facility.state} ${
+                manageFacilityState.facility.zipcode
+              }`}
+            />
+            <FacilityDataItem
+              title="Organization"
+              description={manageFacilityState.facility.org}
+            />
+            <FacilityDataItem
+              title="Organization type"
+              description={manageFacilityState.facility.orgType}
```

Comment:
can we leverage this to translate the org type for users? https://github.com/CDCgov/prime-simplereport/blob/bb8f3a24caa2699abaedfd94a5241b1b02f4d746/frontend/src/app/signUp/Organization/utils.tsx#L10C1-L10C1 🤔

Reply:
oh you mean the user readable version? Yep!

Reply:
Done!